src/index.ts

```diff
@@ -1,3 +1,21 @@
+export type { Types } from '@balena/sbvr-types';
+
+export type Expanded<T> = Extract<T, any[]>;
+export type PickExpanded<T, K extends keyof T = keyof T> = {
+	[P in K]-?: Expanded<T[P]>;
+};
+export type Deferred<T> = Exclude<T, any[]>;
+export type PickDeferred<T, K extends keyof T = keyof T> = {
+	[P in K]: Deferred<T[P]>;
+};
+export interface WebResource {
```

Comment:
Can this be gone since this is a major & this exact same typing comes from sbvr-types (as WebResourceType) ?

Reply:
Yup, this snuck back in during a rebase mistaked = fixed